docs/modular-resources/advanced/create-subtype.md

```diff
@@ -4,76 +4,83 @@ linkTitle: "New API Subtype"
-
-This option is provided for advanced users who are unable to use the modular resource system with the robot deployments.
-
-{{% /alert %}}
-
-You need to define a new resource _subtype_ (and an API for that subtype) if:
-
-- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing [component](/components/) or [service](/services/) subtypes.
+- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing {{< glossary_tooltip term_id="component" text="component" >}} or {{< glossary_tooltip term_id="service" text="service" >}} subtypes.
 - You have a resource that could fit into an existing subtype, but you want to define an API with different methods and messages than the ones in the existing [APIs](/program/apis/) for that subtype.
   For example, you have a [sensor](/components/sensor/) and you want to define a `Calibrate` method.
 
-{{% alert title="Tip" color="tip" %}}
+  {{% alert title="Tip" color="tip" %}}
 
-If you want to use most of an existing API but need just a few other functions, try using the [`DoCommand`](/program/apis/#docommand) endpoint and [extra parameters](/program/use-extra-params/) to add custom functionality to an existing subtype.
+  If you want to use most of an existing API but need just a few other functions, try using the [`DoCommand`](/program/apis/#docommand) endpoint and [extra parameters](/program/use-extra-params/) to add custom functionality to an existing subtype.
 
-{{% /alert %}}
+  {{% /alert %}}
 
 ## Define your new resource API
 
 Viam uses [protocol buffers](https://protobuf.dev/) for API definition.
 
-To define a new subtype, you need to define the methods and messages of the new API in [protobuf](https://github.com/protocolbuffers/protobuf), write code in Python or Go to implement the higher level server and client functions required, and generate all necessary [protobuf module files](https://buf.build/docs/generate/usage/).
+To define a new subtype, you need to define the methods and messages of the new API in [protobuf](https://github.com/protocolbuffers/protobuf), generate all necessary [protobuf module files](https://buf.build/docs/generate/usage/), and write code in Python or Go to implement the higher level server and client functions required.
 The following steps guide you through this process in more detail:
 
 1. Decide whether your subtype is a {{< glossary_tooltip term_id="component" text="component" >}} or a {{< glossary_tooltip term_id="service" text="service" >}}.
    If it provides an interface to control hardware, it is a component.
    If it provides higher-level functionality, it is a service.
 1. Choose a name for your subtype.
-   For example, "gizmo."
-1. Create a directory or repository for your module.
+   For example, "gizmo".
```

Comment:
If it's not monospace, wouldn't it follow normal punctuation guidelines?

Reply:
I am happy to change this to monospace. I do not want to risk people putting periods in the names of their components because they are confused
```suggestion
   For example, `gizmo`.
```